3-networks-dual-svpc/modules/base_env/main.tf

```diff
@@ -36,111 +36,141 @@ locals {
     "bigtable.googleapis.com",
     "binaryauthorization.googleapis.com",
+    "blockchainnodeengine.googleapis.com",
+    "certificatemanager.googleapis.com",
     "cloud.googleapis.com",
+    "cloudaicompanion.googleapis.com",
     "cloudasset.googleapis.com",
     "cloudbuild.googleapis.com",
-    "clouddebugger.googleapis.com",
+    "cloudcontrolspartner.googleapis.com",
     "clouddeploy.googleapis.com",
     "clouderrorreporting.googleapis.com",
     "cloudfunctions.googleapis.com",
     "cloudkms.googleapis.com",
     "cloudprofiler.googleapis.com",
-    "cloudresourcemanager.googleapis.com",
+    "cloudquotas.googleapis.com",
     "cloudscheduler.googleapis.com",
     "cloudsearch.googleapis.com",
+    "cloudtasks.googleapis.com",
     "cloudtrace.googleapis.com",
     "composer.googleapis.com",
     "compute.googleapis.com",
+    "confidentialcomputing.googleapis.com",
+    "config.googleapis.com",
     "connectgateway.googleapis.com",
+    "connectors.googleapis.com",
     "contactcenterinsights.googleapis.com",
     "container.googleapis.com",
     "containeranalysis.googleapis.com",
     "containerfilesystem.googleapis.com",
     "containerregistry.googleapis.com",
+    "containersecurity.googleapis.com",
     "containerthreatdetection.googleapis.com",
+    "contentwarehouse.googleapis.com",
     "datacatalog.googleapis.com",
     "dataflow.googleapis.com",
+    "dataform.googleapis.com",
     "datafusion.googleapis.com",
+    "datalineage.googleapis.com",
     "datamigration.googleapis.com",
+    "datapipelines.googleapis.com",
     "dataplex.googleapis.com",
     "dataproc.googleapis.com",
     "datastream.googleapis.com",
-    "dialogflow.googleapis.com",
+    "discoveryengine.googleapis.com",
     "dlp.googleapis.com",
     "dns.googleapis.com",
     "documentai.googleapis.com",
-    "domains.googleapis.com",
+    "earthengine.googleapis.com",
+    "essentialcontacts.googleapis.com",
     "eventarc.googleapis.com",
     "file.googleapis.com",
+    "financialservices.googleapis.com",
     "firebaseappcheck.googleapis.com",
     "firebaserules.googleapis.com",
     "firestore.googleapis.com",
     "gameservices.googleapis.com",
-    "gkebackup.googleapis.com",
     "gkeconnect.googleapis.com",
     "gkehub.googleapis.com",
+    "gkemulticloud.googleapis.com",
+    "gkeonprem.googleapis.com",
     "healthcare.googleapis.com",
-    "iam.googleapis.com",
     "iamcredentials.googleapis.com",
+    "iap.googleapis.com",
     "iaptunnel.googleapis.com",
+    "identitytoolkit.googleapis.com",
     "ids.googleapis.com",
     "integrations.googleapis.com",
     "kmsinventory.googleapis.com",
     "krmapihosting.googleapis.com",
     "language.googleapis.com",
-    "lifesciences.googleapis.com",
+    "livestream.googleapis.com",
     "logging.googleapis.com",
+    "looker.googleapis.com",
     "managedidentities.googleapis.com",
     "memcache.googleapis.com",
     "meshca.googleapis.com",
-    "meshconfig.googleapis.com",
```

Comment:
`meshconfig.googleapis.com` is supported at GA level
https://cloud.google.com/vpc-service-controls/docs/supported-products

```
Cloud Service Mesh
Status	GA. This product integration is fully supported by VPC Service Controls.
Protect with perimeters?	Yes. You can configure your perimeters to protect this service.
Service name	meshca.googleapis.com,
meshconfig.googleapis.com
Details	
The API for Anthos Service Mesh can be protected by VPC Service Controls, and the product can be used normally inside service perimeters.

You can use mesh.googleapis.com to enable the required APIs for Cloud Service Mesh. You don't need to restrict mesh.googleapis.com in your perimeter as it doesn't expose any APIs.

Learn about configuring private clusters when installing multiple clusters in Cloud Service Mesh.
Learn about adding Anthos Service Mesh services to the service perimeters.
For more information about Cloud Service Mesh, refer to the product documentation.

Limitations	
Service perimeters are not currently supported with Cloud Service Mesh Managed Control Plane.
```